assets/src/blocks/ShareButtons/ShareButtonsBlock.js

```diff
@@ -0,0 +1,83 @@
+import { ShareButtonsEditor } from './ShareButtonsEditor';
+import { ShareButtonsFrontend } from './ShareButtonsFrontend';
+
+const { registerBlockType } = wp.blocks;
+const { __ } = wp.i18n;
+
+const BLOCK_NAME = 'planet4-blocks/share-buttons';
+
+export const registerBlock = () => {
+  return registerBlockType(BLOCK_NAME, {
```

Comment:
You can directly include this in the editor script, like this block does https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/17f159f23992cb58f164fd9c198a1ab07389b64d/assets/src/blocks/Accordion/AccordionEditorScript.js#L35-L56